nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -86,7 +86,7 @@ let
       ''}
 
       ssl_protocols ${cfg.sslProtocols};
-      ssl_ciphers ${cfg.sslCiphers};
+      ${optionalString (cfg.sslCiphers != null) "ssl_ciphers ${cfg.sslCiphers};"}
```

Comment:
I see that we still enable tls1.2 in `sslProtocols`. The issue you linked only refers to tls1.3. Should we therefore not keep `ssl_ciphers`?

Reply:
Ah. Got it now. It will be still used because this is our default setting, but you can now unset it as well.